scipy/stats/_stats_py.py

```diff
@@ -5217,6 +5220,173 @@ def _euclidean_dist(x):
 MGCResult = namedtuple('MGCResult', ('stat', 'pvalue', 'mgc_dict'))
 
 
+def _check_inputs(x,y, compute_distance=_euclidean_dist, reps=1000,
+                         workers=1, is_twosamp=False, random_state=None):
+    r"""
+    _check_inputs takes all inputs from multiscale_graphcorr and 
+    distance_correlation to check each input for any errors. If there are any 
+    errors, the function will raise it and stop the functions from running. 
+    If all inputs are acceptable the check inputs outputs the x and y.
+    
+    Parameters
+    ----------
+    x, y : ndarray
+        If ``x`` and ``y`` have shapes ``(n, p)`` and ``(n, q)`` where `n` is
```

Comment:
MGC references, change to MGC or Dcorr

Reply:
https://github.com/kfangster/scipy/pull/1/commits/e9cbccd5978fcb3b11cfac821bea1d3b18e1f7a0

Reply:
Sorry, this comment was wrong. Since the method applies for both, it should say `MGC and Dcorr`

Reply:
https://github.com/kfangster/scipy/pull/1/commits/c2849f4ae28da79a9df7f4f7aa11339f24385b2a